tests/Kiota.Builder.Tests/Writers/Python/CodeMethodWriterTests.cs

```diff
@@ -25,12 +26,17 @@ public sealed class CodeMethodWriterTests : IDisposable
     private const string MethodDescription = "some description";
     private const string ParamDescription = "some parameter description";
     private const string ParamName = "param_name";
+    private readonly CodeMethodWriter _codeMethodWriter;
+
+
     public CodeMethodWriterTests()
     {
         writer = LanguageWriter.GetLanguageWriter(GenerationLanguage.Python, DefaultPath, DefaultName);
         tw = new StringWriter();
         writer.SetTextWriter(tw);
         root = CodeNamespace.InitRootNamespace();
+        _codeMethodWriter = new CodeMethodWriter(new PythonConventionService(), ClientNamespaceName, false);
```

Comment:
I believe this new variable is still unused. We can probably remove it from the class.